backend/endpoints/sync.py

```diff
@@ -108,22 +108,22 @@ async def get_sidebar(org_id: str, member_id: str, room_type: RoomType):
     Raises:
         HTTPException [dict]: dict containing error message and 400 status code
     """
-    if not ObjectId.is_valid(org_id):
+    if not ObjectId.is_valid(org):
         raise HTTPException(
             status_code=status.HTTP_400_BAD_REQUEST,
-            detail="org_id is not a valid object id",
+            detail="org is not a valid object id",
         )
-    if not ObjectId.is_valid(member_id):
+    if not ObjectId.is_valid(user):
         raise HTTPException(
             status_code=status.HTTP_400_BAD_REQUEST,
-            detail="member_id is not a valid object id",
-        )
-    if room_type is None:
-        raise HTTPException(
-            status_code=status.HTTP_400_BAD_REQUEST, detail="room_type is required"
+            detail="user is not a valid object id",
         )
+   # if room_type is None:
+   #     raise HTTPException(
+   #         status_code=status.HTTP_400_BAD_REQUEST, detail="room_type is required"
+   #     )
 
-    data = await sidebar.format_data(org_id, member_id, room_type)
+    data = await sidebar.format_data(org, user, RoomType.CHANNEL)
```

Comment:
@Bwise1 
This line of code means only data for Channels is going to be recovered.
What about data for DM, because the former code took care of that ?

Reply:
The solution to this is to do two calls, one with `RoomType.CHANNEL` and one with `RoomType.DM`. We will have to discuss how this payload is structured.

Reply:
Alternatively, we can refactor `sidebar.format_data()` to just give the complete payload.